app/controllers/decidim/proposals/proposals_controller.rb

```diff
@@ -74,6 +74,47 @@ def create
         end
       end
 
+      def edit
```

Comment:
Overrides `ProposalsController` because they have customized the comments system and categories. Also we need to override the `Proposal` views. 

Can we extend a controller without modifing the Decidim Original?

Reply:
I tried to extend another Controller using decorator, but it clash with concerns. Somehow the decorator does not loads properly. And a partial overwritting also don't work; you have to copy all controller to make it work.